yarn-project/noir-libs/noir-aztec/src/types/grumpkin_scalar.nr

```diff
@@ -0,0 +1,30 @@
+use crate::types::type_serialisation::TypeSerialisationInterface;
+
+struct GrumpkinScalar {
+    high: Field,
+    low: Field,
+}
```

Comment:
We need a "large-than-field-element" struct in Noir badly. IIRC this is not the first time we need to do something like this.